docs/knowledge-base/DevOps/pipelines.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Pipelines
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 6
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+We store all our helm charts of all the teams under the [helm-charts repository](https://github.com/MapColonies/helm-charts)
+
+This repository is a mono repo, where each team has it’s own `scope`. For example, `3D` team has it’s own directory where the team is storing their helm charts.  
+
+<u>Here’s the repo structure:</u>    
+```bash
+- /
+  - 3D/
+    - Umbrella chart
+    - deployment.json
+    - artifacts.json
+  - raster/
+    - Umbrella chart
+    - deployment.json
+    - artifacts.json
+```
+
+## Release Workflow
+
+We can see the diagram of the Release workflow below
+
+![release workflow diagram](./img/release-workflow-diagram.png)
+
+### CI part
+
+The release workflow begins with a new release or tag push on each service repository and it's happening via Github Actions workflows.
+
+The workflows are being triggered automatically.  
+If we take a look at a sample repo `store-trigger` it has a workflow called `Build and push artifacts` , where the source of this can be found [here](https://github.com/MapColonies/store-trigger/blob/master/.github/workflows/build-and-push.yaml)
+
+The workflow is building the dokcer image of the service, tagging it and pushing it to `Azure ACR`, along with the docker image it's also pushing the helm package which is part of the repository.  
+Afterwards it's updating the `artifacts.json` which is associated with the team's `scope` under the [helm-charts repository](https://github.com/MapColonies/helm-charts) with the new releases which have been just released.  
+
+### CD part
+
+We're utilizing [Jenkins](https://jenkins.mapcolonies.net/) for this purpose.
+
+#### How to login?
```

Comment:
This is relevant to Jenkins in general and should be placed at the top in a preview section.